Patch Testing

samsherwood opened the pull request:
This should trigger a PR build on Gatsby Cloud

Commit:
gitflow-feature-stash: patch-post-install

Commit:
Merge branch 'master' into develop

Commit:
Merge branch 'feature/patch-post-install' into develop

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**Lake-Erie-College-Primary**

:tada: Your build was successful! [See the Deploy preview here.](https://build-63322ca5-680f-4935-b142-33185f39732b.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/9593faa8-b7f3-4af6-89e5-0e41a374ecd3/sites/579ae40e-171e-4342-9df2-d8a7e1799498/builds/63322ca5-680f-4935-b142-33185f39732b/details)

:clock1: Build time: **3m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 44   |
| Accessibility |   :large_orange_diamond: 83   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 82   |

:link: [View full report](https://build-63322ca5-680f-4935-b142-33185f39732b.gtsb.io/reports/lighthouse/index.html)

Commit:
Patches for other React-Spring Packages

Removed sideEffects: false from all associated react-spring packages in hopes of fixing production issue.
